README.md

```diff
@@ -0,0 +1,197 @@
+
+
+### 1. Setting up GCP
+
+First we need to create and setup a GCP project. To do so, you can follow [this guide on Snowplow's wiki](https://github.com/snowplow/snowplow/wiki/GCP:-Getting-Started). Following those instructions, you'll need to enable the following APIs/services:
+
+- Dataflow
+- Bigtable
+- Pub/Sub
+- Storage
+
+After that, you'll need to authenticate the computer where you'll be running the project. To do so, run the following commands:
+
+```bash
+$ gcloud auth login
+$ gcloud auth application-default login
+```
+
+Note: Some of the following tasks can be performed using the helper script provided in this repo. The script uses pyinvoke - each action provided by the script is preceded by 'inv'. Run ``` $ inv --list ``` on the repo root to list the available actions on the script. 
+
+Note 2: At some important steps the script assumes default configurations, which might not be the right ones for you. For example, when creating a bigtable instance and table, the script will choose SSD storage over HDD, which is much more expensive. 
+
+#### Cloud Pub/Sub
+
+After enabling Pub/Sub, go to https://console.cloud.google.com/cloudpubsub?project=YOUR-PROJECT-ID . Note that you have to change *YOUR-PROJECT-ID* to, you guessed it, your project id.
+
+There, click "Create Topic" and type in whatever name you want to give to the topic we'll be using to read the events from. In our example, we used "test-topic". Keep track of your topic's name (you can always go to that URL to check it), as it will be needed when you're creating your config file.
+
+![creating a pubsub topic][create-topic]
+create a staging location in which the system will store the jars ran by the workers. To do this, you need to enable the Storage API.
+
+After that, go to https://console.cloud.google.com/storage/browser?project=YOUR-PROJECT-ID . Click "Create Bucket". Fill in the appropriate details. As this is an example project, we suggest you pick the cheapest option.
+
+After creating your bucket, you might want to create a folder inside it, to serve as your staging location. You can also just use the bucket's root. To create a folder in your bucket, after you created the bucket, select it in the list and then click "Create Folder".
+
+Your staging location will be:
+
+```bash
+gs://your-bucket-name/the-folder-that-you-might-have-created
+```
+
+![setting staging location][bucket1]
+
+![setting staging location][bucket2]
+
+#### Bigtable
+
+After enabling Bigtable, we'll need to create an instance where our table will be stored. And then create the actual table.
+
+##### Creating an instance
+
+Go to https://console.cloud.google.com/bigtable/instances?project=YOUR-PROJECT-ID , again taking care to change it with your project id. Click "Create instance" and fill the appropriate details. The name is, again, important. We used "test-instance" in our example. We suggest you to select HDD as a storage type as it is much cheaper, and this is for test-purposes only.
```

Comment:
taking care of changing it

fill in the appropriate details

as storage type

testing purposes